src/content/en/updates/2018/07/css-scroll-snap.md

```diff
@@ -0,0 +1,365 @@
+       target="_blank">View demo</a> |
+    <a href="https://glitch.com/edit/#!/snap?path=product.html:1:0"
+       target="_blank">Source</a>
+  </figcaption>
+</figure>
+
+## Interaction with other scrolling APIs
+
+### DOM Scrolling API {#dom-scrolling-api}
+
+Scroll snapping happens **after** all scroll operations including those
+initiated by script. When you are using APIs like `Element.scrollTo`, the
+browser will calculate the intended scroll position of the operation, then apply
+appropriate snapping logic to find the final snapped location. Thus,  there is
+no need for user script to do any manual calculations for snapping.
+
+
+### Smooth Scrolling  {#smooth-scrolling}
+
+Smooth scrolling controls the behavior of a programmatic scroll operation while
+scroll snap determines its destination. Since they control orthogonal aspects of
+scrolling, they can be used together and complement each other.
+
+
+### Overscroll Behavior {#overscroll-behavior}
+
+[Overscroll behavior
+API](https://developers.google.com/web/updates/2017/11/overscroll-behavior)
+controls how scroll is chained across multiple elements and it is not affected
+by scroll snap.
+
+
+## Caveats and best practices
+
+
+Avoid using mandatory snapping when target elements are widely spaced apart.
+This can cause content in between the snap positions to become inaccessible.
+
+Use `CSS.supports` for feature detecting CSS Scroll Snap.  But avoid using
+`scroll-snap-type` which is also present in the deprecated specification and can
+be unreliable.
+
+
+```
+if (CSS.supports('scroll-snap-align: start')) {
+  // use css scroll snap
+} else {
+  // use fallback
+}
+```
+
+
+Do not assume that programmatically scrolling APIs such as `Element.scrollTo`
+always finish at the requested scroll offset. Scroll snapping may adjust the
+scroll offset after programmatic scrolling is complete. Note that this was not a
+good assumption even before scroll snap since scrolling may have been
+interrupted for other reasons but it is especially the case with scroll
+snapping.
+
+
+<aside>
+  <strong>Note:</strong>
+  There is an <a href="https://github.com/w3c/csswg-drafts/issues/1562#issuecomment-389586317">
+  upcoming proposal</a> to change various scrolling APIs to return a promise.
+  This promise is resolved when user agent either completes or aborts that
+  scrolling operation. Once this is standardized and implemented, it provides an
+  ergonomic and efficient way for following up a user script initiated scroll
+  with other actions.
+</aside>
+
+
+## Future work
+
+Chrome 69 ships the core functionality specified in CSS Scroll Snap
+specification. The main omissions are snapping for keyboard scrolling and
+fragment navigations which at the moment are not supported by any other
+implementations. Chrome will continue improving this feature over time
+particularly focusing on missing features, improving snap selection algorithm,
+animation smoothness, and devtools facilities.
```

Comment:
Please add `'{% include "web/_shared/rss-widget-updates.html" %}` to the bottom of the file. Optionally, you can add `{% include "comment-widget.html" %}` if you want to enable comments.

Reply:
Added both. No reason to disable comments.